scripts/validation/__tests__/validate-branch-name.test.js

```
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const { spawnSync } = require("child_process");
```

ESLint

[error] 'require' is not defined.

(no-undef)